src/ResponseType.php

```diff
@@ -127,6 +127,14 @@ public static function getAll(): array {
 				$binaryType,
 				null,
 			),
+			new ResponseType(
+				'StreamGeneratorResponse',
+				false,
+				false,
+				null,
+				$binaryType,
```

Comment:
```suggestion
				null,
				null,
```
Since there is no default content type there can't be a default type either.

Reply:
Done